src/applications/disability-benefits/2346/config/form.js

```diff
@@ -1,22 +1,38 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
```

Comment:
Why are you disabling these ESLint rules?

Reply:
Just removed these comments.